samples/test/automlVisionObjectDetectionDataset.v1beta1.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright 2019, Google LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const {assert} = require('chai');
+const execa = require('execa');
+
+/** Tests for AutoML Vision Object Detection "Dataset API" sample. */
+
+const cmdDataset = 'node automlVisionObjectDetectionDataset.js';
+
+// TODO(developer): Before running the test cases,
+// set the environment variables PROJECT_ID, REGION_NAME and
+// change the value of datasetId
+const projectId = process.env.PROJECT_ID;
```

Comment:
Instead let's use `await client.getProjectId()`

Reply:
this is a test file. i'm assuming this doesn't need fixing.